src/wp-includes/fonts/class-wp-font-library.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * Font Library class.
+ *
+ * This file contains the Font Library class definition.
+ *
+ * @package    WordPress
+ * @subpackage Font Library
+ * @since      6.5.0
+ */
+
+/**
+ * Font Library class.
+ *
+ * @since 6.5.0
+ */
+class WP_Font_Library {
+
+	/**
+	 * Provide the expected mime-type value for font files per-PHP release. Due to differences in the values returned these values differ between PHP versions.
+	 *
+	 * This is necessary until a collection of valid mime-types per-file extension can be provided to 'upload_mimes' filter.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param array $php_version_id The version of PHP to provide mime types for. The default is the current PHP version.
+	 *
+	 * @return Array A collection of mime types keyed by file extension.
+	 */
+	public static function get_expected_font_mime_types_per_php_version( $php_version_id = PHP_VERSION_ID ) {
```

Comment:
I think I flagged this last time already, I don't think this method should be public.

Testability is not an argument in this case. This can be tested indirectly if needed.

Reply:
Oh, and I also flagged that it makes zero sense to pass the current PHP version as a parameter. In #5285 that part was at least fixed.

Why are we reintroducing this again?

Reply:
I agree and I just realized that set_mimetypes shouldn't exist as well, I'll fix these. (I was not aware of the previous review)

Reply:
It should be better now.